src/components/Header.tsx

```diff
@@ -13,24 +14,30 @@ const HeaderContainer = styled.div`
   width: 100%;
 `;
 
-export interface HeaderProps {
-  lastError: { message: string };
-  onCloseError: () => void;
-}
-
-export const Header = (props: HeaderProps) => {
+export const Header = () => {
+  const messageContext = useContext(MessageContext);
+  const messages = messageContext.messages;
+  console.log("Rendering HEader with: ", messages);
```

Comment:
```suggestion
  console.log("Rendering Header with: ", messages);
```

Reply:
I removed this log. I used it only once for debugging and forgot about it.